### ISSUE TITLE
Correctly parse expressions like `TableA.*` in the postgresparser

### ISSUE THREAD

**Issue by mush-zhang** [cmu-db/noisepage#478]
Terrier and Peloton does not handle such expression. For example, `SELECT TableA.* FROM TableA` would parse the `TableA.*` into a ColumnValueExpression with database name "TableA" and column name some weird string. 

Should we parse it into a ColumnValueExpression with database name "TableA" and its child a StarExperssion?

**Comment by apavlo** [comment]
Sample database:

```sql
CREATE TABLE A (a1 INT PRIMARY KEY, a2 INT);
CREATE TABLE B (b1 INT PRIMARY KEY, b2 INT);
INSERT INTO A VALUES (1, 1), (2, 2), (3, 3), (4, 4);
INSERT INTO B VALUES (1, 1), (2, 2), (3, 3), (4, 4);
```

Using `*` by itself in the output list works. But adding the tablename doesn't work. This output is from the old Peloton code:
```
xxx=# SELECT A.* FROM A;
Cannot find column y
xxx=# SELECT B.*, A.* FROM A JOIN B ON A.a1 = B.b1;
Cannot find column y
xxx=# SELECT * FROM A JOIN B ON A.a1 = B.b1;
 b1 | b2 | a1 | a2 
----+----+----+----
  1 |  1 |  1 |  1
  2 |  2 |  2 |  2
  3 |  3 |  3 |  3
  4 |  4 |  4 |  4
(4 rows)
```

**Comment by lmwnshn** [comment]
Fixed by #1248.
